sphinx/source/docs/user_guide/server.rst

```diff
@@ -1052,6 +1052,42 @@ As before, run the Bokeh server with the following command:
 
     bokeh serve myapp.py --port 5100
 
+.. _userguide_server_deployment_unix_socket:
+
+Restricting access to proxied server using unix socket
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Sphinx is really picky. This underline needs to extend exactly under the text above.